styles/layout/column.css

```diff
@@ -0,0 +1,47 @@
+@media only screen and ( min-width: 1440px ) {
```

Comment:
I've chosen the breakpoint between the mobile and desktop view to be 1440 btw, but you could also pick 1024 if you'd want. 768 becomes a bit risky already, so I wouldn't go lower than 1024 but its up to you to check it it out